src/main/resources/youtube/settings/host/values/strings.xml

```diff
@@ -1840,7 +1839,7 @@ Tap on the continue button and disable battery optimizations."</string>
 • Stable volume is not available."</string>
     <string name="revanced_spoof_streaming_data_side_effects_unknown">• Video may not play.</string>
     <string name="revanced_spoof_streaming_data_ios_compatibility_title">iOS Compatibility mode</string>
-    <string name="revanced_spoof_streaming_data_ios_compatibility_summary_on">Only spoofed as iOS client if it is not a movie, paid video, or livestream.</string>
```

Comment:
[As I commented](https://github.com/inotia00/revanced-integrations/pull/59#discussion_r1843429007), the code in the PR never runs

In the livestream, it still spoofs the Android VR client or Android TV client, not the iOS client